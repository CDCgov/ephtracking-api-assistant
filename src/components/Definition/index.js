import React, { Fragment } from 'react';
import PropTypes from 'prop-types';

const Definition = ({ children }) => (
  <Fragment>
    <h5 className="title is-5">Definition</h5>
    <p>{children}</p>
  </Fragment>
);

Definition.propTypes = {
  children: PropTypes.node.isRequired
};

export default Definition;
