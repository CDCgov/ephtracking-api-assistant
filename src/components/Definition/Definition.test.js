import React from 'react';
import Definition from './';
import renderer from 'react-test-renderer';

describe('Defintion Component', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<Definition>this is a definition</Definition>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
