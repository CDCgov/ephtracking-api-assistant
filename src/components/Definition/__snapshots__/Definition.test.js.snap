// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <h5
    className="title is-5"
>
    Definition
</h5>,
  <p>
    this is a definition
</p>,
]
`;
